Autocoders/Python/src/fprime_ac/generators/visitors/ComponentVisitorBase.py

```diff
@@ -379,12 +379,10 @@ def g(xxx_todo_changeme3):
                 if len(opcodes) == 1:
                     return f"CMD_{mnemonic.upper()}"
                 else:
-                    mlist = []
-                    inst = 0
-                    for opcode in opcodes:
-                        mlist.append("CMD_" + mnemonic.upper() + "_%d" % inst)
-                        inst += 1
-                    return mlist
+                    return [
+                        f"CMD_{mnemonic.upper()}" + "_%d" % inst
```

Comment:
Shouldn't this become:

f"CMD_{mnemonic.upper()}_{inst}"

Reply:
Indeed, thanks!